FeedAPIChallenge/Feed API/RemoteImage.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  RemoteImage.swift
+//  FeedAPIChallenge
+//
+//  Created by Daniel Gallego Peralta on 13/6/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal struct RemoteImage: Decodable {
```

Comment:
The convention in this codebase is to omit the default access level of `internal`, please remove it.
```suggestion
struct RemoteImage: Decodable {
```